### DISCUSSION TITLE
In map visual/audio/input settings panel behaviour change

### DISCUSSION THREAD

**Discussion by LosPedros** [ppy/osu#30733] [Ideas]
The visual/audio/input settings which show at the start of the map should have a way to change them mid map.

It's something stable has when putting the cursor to the bottom of the screen when paused but lazer doesn't, it is annoying as I do sometimes change these settings mid gameplay, would be cool if putting my cursor in the top right corner pulled the panel to view mid map when paused.

**Comment by lunarcosmix** [comment]
Agree with this especially since they were added to multi spectator/replay in #30749. It would help people coming from stable adjust to per-map settings; without restarting or searching through the main settings (using Ctrl+O)
